src/components-examples/material/tree/tree-checklist/tree-checklist-example.ts

```diff
@@ -171,9 +171,9 @@ export class TreeChecklistExample {
   /** Whether all the descendants of the node are selected. */
   descendantsAllSelected(node: TodoItemFlatNode): boolean {
     const descendants = this.treeControl.getDescendants(node);
-    const descAllSelected = descendants.every(child =>
-      this.checklistSelection.isSelected(child)
-    );
+    const descAllSelected = descendants.length > 0 && descendants.every(child => {
+      return this.checklistSelection.isSelected(child);
```

Comment:
nit: why add the `{}` and `return`?

Reply:
I just find it more readable compared to the multi-line arrow function with an implicit return.

Reply:
I prefer the implicit return, but I don't think we have an official style on way or the other